packages/application-generic/src/utils/filter-processing-details.ts

```diff
@@ -1,7 +1,11 @@
-import { StepFilter } from '@novu/dal';
-import { ICondition } from '@novu/shared';
+import { StepFilter, SubscriberEntity } from '@novu/dal';
+import { ICondition, ITriggerPayload } from '@novu/shared';
 
-import { IFilterVariables } from './types';
+export interface IFilterVariables {
```

Comment:
Might need to add tenant here as well. Like when we have the whole subscriber entity, we might need the tenant entity as well.

Reply:
I do that in my next pr